src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
The `ColorSupplier.getRandomColor()` method call should be using an instance of `ColorSupplier` rather than calling a static method. Static methods are generally considered bad practice in this context. You should create an instance of `ColorSupplier` and call the non-static method on that instance.